File: messaging/v1/short_code/create-default/create-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

short_code = @client.messaging
                    .services('MGXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                    .short_codes
                    .create(
                       short_code_sid: 'SCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
                     )

puts short_code.sid

File: voice/v1/bulk_country_update/create-default/create-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Voice.V1.VoicePermission;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var bulkCountryUpdate = BulkCountryUpdateResource.Create(
            updateRequest: "UpdateRequest"
        );

        Console.WriteLine(bulkCountryUpdate.UpdateCount);
    }
}
